scripts/run.sh

```diff
@@ -8,7 +8,13 @@
 set -e
 
 # Prefer possible `cargo build` binaries over PATH binaries
-cd "$(dirname "$0")/.."
+script_dir="$(realpath "$(dirname "$0")")"
```

Comment:
```suggestion
script_dir="$(readlink -f "$(dirname "$0")")"
```

Neither of these is compatible with BSD coreutils on macos, but at least `readlink -f` is consistent with the rest of the shell scripts in the monorepo